### PR TITLE
fix: console error with settings group in case of RTL languages.

### PR DESCRIPTION
**Pull request by shubhw12**
### Description
In the case of RTL languages, we had an issue for the UI tab key being passed the same as the language. fixed this by setting a counter and based on that counter the key will be decided.

### How has this been tested?

Checked with 3 color options and in RTL languages and LTR languages as well.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js [hunk 1]
```diff
@@ -1384,15 +1384,21 @@ wp.customize.controlConstructor['ast-responsive-background'] = wp.customize.Cont
...
 
-                _.each( fields.tabs, function ( value, key ) {
-
+                _.each( fields.tabs, function ( value , key ) {
                     var li_class = '';
                     if( 0 == counter ) {
                         li_class = "active";
+                        tab_key = 'normal';
+                    }
+                    else if( 1 == counter ){
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:1395]: @shubhw12 Please use Yoda conditions here.

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js [hunk 2]
```diff
@@ -1401,8 +1407,17 @@ wp.customize.controlConstructor['ast-responsive-background'] = wp.customize.Cont
                 fields_html += '<div class="ast-tab-content" >';
 
                 _.each( fields.tabs, function ( fields_data, key ) {
+                    if( 0 == tabs_counter ) {
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:1410]: @shubhw12 Please use Yoda conditions here.

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js [hunk 3]
```diff
@@ -1384,15 +1384,21 @@ wp.customize.controlConstructor['ast-responsive-background'] = wp.customize.Cont
                 fields_html += '<div id="' + clean_param_name + '-tabs" class="ast-group-tabs">'; 
                 fields_html += '<ul class="ast-group-list">'; 
                 var counter = 0;
+                var tabs_counter = 0;
 
-                _.each( fields.tabs, function ( value, key ) {
-
+                _.each( fields.tabs, function ( value , key ) {
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:1389]: @shubhw12 Please remove extra space before comma between value and key.

File: inc/customizer/custom-controls/settings-group/settings-group.js [hunk 4]
```diff
@@ -140,15 +140,21 @@
                 fields_html += '<div id="' + clean_param_name + '-tabs" class="ast-group-tabs">'; 
                 fields_html += '<ul class="ast-group-list">'; 
                 var counter = 0;
+                var tabs_counter = 0;
 
-                _.each( fields.tabs, function ( value, key ) {
-
+                _.each( fields.tabs, function ( value , key ) {
```
Comment (premanshup) [inline inc/customizer/custom-controls/settings-group/settings-group.js:145]: @shubhw12 Please remove extra space before comma between value and key.

**Review by premanshup — CHANGES_REQUESTED** [review]

**Review by premanshup — APPROVED** [review]

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js (hunk 3 above)
Comment (shubhw12) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js]: Donw

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js (hunk 2 above)
Comment (shubhw12) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js]: Done

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js [hunk 5]
```diff
@@ -1401,8 +1407,17 @@ wp.customize.controlConstructor['ast-responsive-background'] = wp.customize.Cont
                 fields_html += '<div class="ast-tab-content" >';
 
                 _.each( fields.tabs, function ( fields_data, key ) {
+                    if( 0 === tabs_counter ) {
```
Comment (imnavanath) [inline inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:1410]: We can replace if-else-if logic with switch case.

Comment (shubhw12) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js]: Yes I have replaced it with switch case.

**Review by imnavanath — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

**Pull request merged**